code/game/objects/items/weapons/twohanded.dm

```diff
@@ -442,3 +442,47 @@
 			M.faction = user.faction.Copy()
 			M.Copy_Parent(user, 100, user.health/2.5, 12, 30)
 			M.GiveTarget(L)
+
+
+
+
+/obj/item/weapon/twohanded/pitchfork
+	icon_state = "pitchfork0"
+	name = "pitchfork"
+	desc = "A red pitchfork, it looks like the work of the devil."
```

Comment:
Make it a child of pitchfork, so that we can have normal metal pitchfork for ~~angry mobs~~ botanists.